lib/chai/core/assertions.js

```diff
@@ -221,6 +222,17 @@ module.exports = function (chai, _) {
    *     expect([1,2,3]).to.include(2);
    *     expect('foobar').to.contain('foo');
    *     expect({ foo: 'bar', hello: 'universe' }).to.include({ foo: 'bar' });
+   * 
+   * By default, strict equality (===) is used.
```

Comment:
Maybe this line isn't totally clear for anyone using the library.
When saying that this uses `strict` equality this may not be exactly clear for everyone. This is because when saying that this assertion uses this kind of comparison I would expect (if I just read the docs) the following assertion to throw an error:

```js
expect({foo: obj1, bar: obj2}).to.include({foo: obj1});
```

Because **I would expect it to compare the object I passed as a parameter to the include method to the one I passed to `expect`, and, since they would not be the same instance, strict equality would be false** and therefore I would expect an `Error` to be raised.

Maybe we could explain this a little better, just to make things as clear as possible, I'm not sure everyone will get it right this way.

What do you guys think?